apps/mobile/src/icons/EmailIcon.tsx

```diff
@@ -0,0 +1,10 @@
+import Svg, { Path, SvgProps } from 'react-native-svg';
```

Comment:
oo we have an EmailIcon already, looks like it's in screens/Login/EmailIcon

I think this /icons/ folder is better though, would you be able to consolidate these?

Reply:
Yup! I've deleted the new files and consolidate the icon into the `icons` folder